lib/views/flight.py

```diff
@@ -1,109 +1,159 @@
-    flight_inclination: str
-    flight_heading: str
-
-
-class OutOfRailConditions(BaseModel):
-    out_of_rail_time: str
-    out_of_rail_velocity: str
-    out_of_rail_static_margin: str
-    out_of_rail_angle_of_attack: str
-    out_of_rail_thrust_weight_ratio: str
-    out_of_rail_reynolds_number: str
-
-
-class BurnoutConditions(BaseModel):
-    burnout_time: str
-    burnout_rocket_velocity: str
-    burnout_altitude: str
-    burnout_freestream_velocity: str
-    burnout_mach_number: str
-    burnout_kinetic_energy: str
-
-
-class ApogeeConditions(BaseModel):
-    apogee_time: str
-    apogee_altitude: str
-    apogee_freestream_speed: str
-
-
-class MaximumValues(BaseModel):
-    maximum_speed: str
-    maximum_mach_number: str
-    maximum_reynolds_number: str
-    maximum_dynamic_pressure: str
-    maximum_acceleration_during_motor_burn: str
-    maximum_acceleration_after_motor_burn: str
-    maximum_gs_during_motor_burn: str
-    maximum_gs_after_motor_burn: str
-    maximum_upper_rail_button_normal_force: str
-    maximum_upper_rail_button_shear_force: str
-    maximum_lower_rail_button_normal_force: str
-    maximum_lower_rail_button_shear_force: str
-
-
-class ImpactConditions(BaseModel):
-    x_impact_position: "Optional[str]"
-    y_impact_position: "Optional[str]"
-    time_of_impact: "Optional[str]"
-    impact_velocity: "Optional[str]"
-
-
-class EventsRegistered(BaseModel):
-    events_trace: "Optional[Any]"
-
-
-class FlightData(BaseModel):
-    initial_conditions: InitialConditions
-    numerical_integration_settings: NumericalIntegrationSettings
-    launch_rail_conditions: LaunchRailConditions
-    surface_wind_conditions: SurfaceWindConditions
-    out_of_rail_conditions: OutOfRailConditions
-    burnout_conditions: BurnoutConditions
-    apogee_conditions: ApogeeConditions
-    maximum_values: MaximumValues
-    impact_conditions: ImpactConditions
-    events_registered: "Optional[EventsRegistered]"
-
-
-class FlightPlots(BaseModel):
-    pass
-
-
-class FlightSummary(BaseModel):
-    flight_data: FlightData
-    # flight_plots: FlightPlots
+from lib.models.flight import Flight
+from lib.views.rocket import RocketView, RocketSummary
+from lib.views.environment import EnvSummary
+
+
+class FlightSummary(RocketSummary, EnvSummary):
+    # TODO: if Any is Callable, jumps pydantic parsing, expects a dill binary object
+    # TODO: implement {flight_id}/summary/motor; {flight_id}/summary/rocket; {flight_id}/summary/environment
+    name: Optional[str]
+    max_time: Optional[int]
+    min_time_step: Optional[int]
+    # max_time_step: Optional[float]
+    equations_of_motion: Optional[str]
+    heading: Optional[int]
+    inclination: Optional[int]
+    initial_solution: Optional[list]
+    effective_1rl: Optional[float]
+    effective_2rl: Optional[float]
+    out_of_rail_time: Optional[float]
+    out_of_rail_time_index: Optional[int]
+    parachute_cd_s: Optional[float]
+    post_processed: Optional[bool]
```

Comment:
This is legacy code

```suggestion
```

Reply:
patched in https://github.com/RocketPy-Team/Infinity-API/pull/25/commits/0f0cbd5f168ad455a8255244a73ed6f43a08c87f